<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>611</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>681</width>
    <height>611</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Biblioteczka</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2e2e2e;
    color: #ffffff;
	margin: 10px;
}

QGridLayout {
    background-color: #2e2e2e;
    color: #ffffff;
    margin: 10px; /* Dodaje margines 10 pikseli dookoła grida */
}

QPushButton {
    background-color: #4a4a4a;
    color: #ffffff;
    border: 1px solid #5a5a5a;
    padding: 5px;
    border-radius: 3px;
}

QPushButton:hover {
    background-color: #5a5a5a;
}

QPushButton:pressed {
    background-color: #3a3a3a;
}

QWidget {
    background-color: #2e2e2e;
    color: #ffffff;
}

QGridLayout {
    background-color: #2e2e2e;
    color: #ffffff;
}

QLabel {
    color: #ffcc00; 
}

QLineEdit {
    background-color: #3a3a3a;
    color: #ffffff;
    border: 1px solid #5a5a5a;
    padding: 5px;
    border-radius: 3px;
}

QLineEdit:focus {
    border: 1px solid #ffcc00; 
}

QComboBox {
    background-color: #3a3a3a;
    color: #ffffff;
    border: 1px solid #5a5a5a;
    padding: 5px;
    border-radius: 3px;
}

QComboBox QAbstractItemView {
    background-color: #3a3a3a;
    color: #ffffff;
    selection-background-color: #5a5a5a;
}

QStatusBar {
    background-color: #2e2e2e;
    color: #ffcc00; 
}

QTableView {
    background-color: #2e2e2e;
    color: #ffffff;
    gridline-color: #5a5a5a;
    border: 1px solid #5a5a5a;
}

QHeaderView::section {
    background-color: #4a4a4a;
    color: #ffffff;
    padding: 5px;
    border: 1px solid #5a5a5a;
}

QTableView::item:selected {
    background-color: #ffcc00;
    color: #2e2e2e;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>651</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="addBookButton">
       <property name="text">
        <string>Dodaj książkę</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="deletePersonButton">
       <property name="text">
        <string>Usuń osobę</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QTableView" name="personsTableView"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="deleteRentButton">
       <property name="text">
        <string>Usuń wypożyczenie</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="deleteBookButton">
       <property name="text">
        <string>Usuń książkę</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" rowspan="2">
      <widget class="QTableView" name="rentalsTableView"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="addPersonButton">
       <property name="text">
        <string>Dodaj osobę</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="addRentButton">
       <property name="text">
        <string>Dodaj wypożyczenie</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QTableView" name="booksTableView"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
